website_demo/index.html

```diff
@@ -0,0 +1,326 @@
+                            <a class="nav-toggle" href="#"><span></span></a>
+                        </div>
+                    </div>
+                </header>
+            </section>
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-10 col-md-offset-1">
+                        <div class="hero-content text-center">
+                            <img src="img/HCPLogoText-Crop.png">
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-sm-4 hidden-xs"></div>
+                    <div class="col-sm-4 hidden-xs text-center"><a href="#rsvp"
+                            class="btn btn-accent btn-large rsvp-btn">Join our mailing list</a></div>
+                    <div class="col-sm-4 hidden-xs"></div>
+                </div>
+            </div>
+            <div class="down-arrow floating-arrow"><a href="#invitation"><i class="fa fa-angle-down"></i></a></div>
+        </section>
+
+        <section id="invitation" class="section-padding">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-12 text-center">
+                        <!-- <h3>We are here, to support our first responders</h3> -->
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section id="intro">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-12 text-center">
+                        <!-- this section is going to be about our project as a whole -->
+                        <h3 class="header">Why we created this project</h3>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-md-6">
+                        <p>
+                            blob about why we created this project
+                        </p>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-sm-4 col-sm-push-2 col-xs-6 hidden-md hidden-lg"><img src="img/IMG_2317.jpg"
+                            class="wp8"></div>
+                    <div class="col-sm-4 col-sm-push-2 col-xs-6 hidden-md hidden-lg"><img src="img/DSD_0214.jpg"
+                            class="wp9"></div>
+                </div>
+            </div>
+        </section>
+
+        <section class="events section-padding" id="events">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-12">
+                        <h3 class="header">Demo</h3>
+                    </div>
+                </div>
+                <div>
+                    <A HREF="../templates/index.html">About this site</A>
+                </div>
+            </div>
+
+            <div id="dc-modal" class="modal fade" tabindex="-1" role="dialog">
```

Comment:
Could you remove this modal (entire div)? This doesn't seem to appear & relevant to our project.